packages/graphql-transformers-e2e-tests/resources/jsonServer/package.json

```diff
@@ -3,12 +3,12 @@
   "private": true,
   "version": "1.0.0",
   "devDependencies": {
-    "aws-cdk": "~2.80.0",
-    "@types/node": "^12.12.6",
-    "typescript": "^3.8.3"
+    "aws-cdk": "~2.129.0",
+    "@types/node": "*",
+    "typescript": "*"
```

Comment:
Is it safe to use just any version here?

Reply:
This local json server gets deployed as part of HttpTransformer test. It is okay here so we test with latest and if there are any breaking changes in future we can detect and fix them appropriately.